frinx-uniconfig/user-guide/uniconfig-operations/uniconfig-properties/rpc_update-properties/readme.md

```diff
@@ -0,0 +1,217 @@
+property value from the database, so UniConfig instances will use different values for the same property.
+
+Recommendation: use this RPC with *UniConfig Cloud Config*. 
+The exception is *callbacks.access-token* which is always up-to-date.
+!!!
+
+## RPC Examples
+
+### Successful Example
+
+RPC input contains the default properties with correct values.
+
+```bash RPC Request
+curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:update-properties' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+       "properties-map": [
+            {
+                "key": "netconf-default-parameters.session-timers.between-attempts-timeout",
+                "value": "1500"
+            },
+            {
+                "key": "gnmi-default-parameters.session-timers.internal-transaction-timeout",
+                "value": "1000"
+            },
+            {
+                "key": "notifications.kafka.transaction-notifications-enabled",
+                "value": "false"
+            }
+       ]
+    }
+}'
+```
+
+```json RPC Response, Status: 200
+{
+  "output": {
+    "update-properties-status": "There are 3 updated and 0 inserted properties successfully. Ignored keys: []"
+  }
+}
+```
+
+### Failed example
+
+RPC input contains the default property crypto.
+
+```bash RPC Request
+curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:update-properties' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+       "properties-map": [
+            {
+                "key": "crypto.encrypt-enabled",
+                "value": "true"
+            },
+            {
+                "key": "crypto.actual-encryption-public-key",
+                "value": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+            }
+       ]
+    }
+}'
+```
+
+```json RPC Response, Status: 200
+{
+  "output": {
+    "update-properties-status": "There are 0 updated and 0 inserted properties successfully. Ignored keys: [crypto.encrypt-enabled, crypto.actual-encryption-public-key]"
+  }
+}
+```
+
+### Failed example
+
+RPC input contains the default properties with incorrect values.
+
+```bash RPC Request
+curl --location --request POST 'http://localhost:8181/rests/operations/uniconfig-manager:update-properties' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+    "input": {
+       "properties-map": [
+            {
+                "key": "netconf-default-parameters.session-timers.between-attempts-timeout",
+                "value": "1500"
+            },
+            {
+                "key": "gnmi-default-parameters.session-timers.internal-transaction-timeout",
+                "value": "true"
+            }
+       ]
+    }
+}'
+```
+
+```json RPC Response, Status: 200
```

Comment:
we should not return 200 if something fails

Reply:
done